README/MATH/Prediction_Market_Math.tex

```diff
@@ -0,0 +1,1763 @@
+
+\[
+C_i = d_i \times b_i
+\]
+
+\subsection*{Example Calculations for Third Transaction}
+
+\textbf{Given:}
+
+\begin{itemize}
+  \item Resolution Probability: \( R = 0.5 \)
+  \item Bet 0: Neutral, \( b_0 = 10 \) units, \( p_0 = 0.5 \)
+  \item Bet 1: NO, \( b_1 = 20 \) units, \( p_1 = 0.167 \)
+  \item Bet 2: YES, \( b_2 = 10 \) units, \( p_2 = 0.375 \)
+  \item Bet 3: YES, \( b_3 = 10 \) units, \( p_3 = 0.5 \)
+\end{itemize}
+
+\textbf{For Bet 1:}
+
+\[
+d_1 = |R - p_1| = |0.5 - 0.167| = 0.333
+\]
+\[
+C_1 = d_1 \times b_1 = 0.333 \times 20 = 6.660 \text{ units}
+\]
+
+\textbf{For Bet 2:}
+
+\[
+d_2 = |R - p_2| = |0.5 - 0.375| = 0.125
+\]
+\[
+C_2 = d_2 \times b_2 = 0.125 \times 10 = 1.250 \text{ units}
+\]
+
+\textbf{For Bet 3:}
+
+\[
+d_3 = |R - p_3| = |0.5 - 0.5| = 0
+\]
+\[
+C_3 = d_3 \times b_3 = 0 \times 10 = 0 \text{ units}
+\]
+
+\begin{center}
+\begin{tikzpicture}[scale=0.5]
+
+% Bet 1: NO Bet (Red Squares), starting at x = 5
+\foreach \x in {5,...,14}
+{
+    \draw[fill=red!50] (\x,0) rectangle ++(1,1);
+    \draw[fill=red!50] (\x,-1) rectangle ++(1,1);
+}
+
+% Label for Bet 1
+\node[above] at (11,1.2) {Bet 1: NO (\( b_1 = 20 \) units at \( p_1 = 0.167 \))};
+
+% Course Payout for Bet 1 (Blue Squares)
+\foreach \x in {-8,...,-3}
+{
+    \draw (\x,0) rectangle ++(1,1);
+    \filldraw[fill=blue!50] (\x,0) rectangle ++(1,1);
+}
+
+% Course Payout for Bet 1 (Blue Squares)
+\def\xPartial{-2}
+\draw (\xPartial,0) rectangle ++(1,1);
+\filldraw[fill=blue!50] (\xPartial,0) rectangle ++(1,0.66);
+
+% Label for Course Payout of Bet 1
+\node[above] at (-4,1.2) {Course Payout for Bet 1 (\( C_1 = 6.660 \) units)};
+
+% Bet 2 YES (Green Squares)
+\foreach \x in {5,...,14}
+{
+    \draw[fill=green!50] (\x,-5) rectangle ++(1,1);
+}
+
+% Label for Bet 2
+\node[above] at (11.2,-4) {Bet 2: YES (\( b_1 = 10 \) units at \( p_1 = 0.375 \))};
+
+
+% Course Payout for Bet 2 (Blue Squares)
+\foreach \x in {-3,...,-3}
+{
+    \draw (\x,-5) rectangle ++(1,1);
+    \filldraw[fill=blue!50] (\x,-5) rectangle ++(1,1);
+}
+
+% Partial Course Payout for Bet 2 (Blue Squares)
+\def\xPartial{-2}
+\draw (\xPartial,-5) rectangle ++(1,1);
+\filldraw[fill=blue!50] (\xPartial,-5) rectangle ++(1,0.25);
+
+% Label for Course Payout of Bet 2
+\node[above] at (-4,-4) {Course Payout for Bet 2 (\( C_1 = 1.25 \) units)};
+
+
+% Bet 3 YES (Green Squares)
+\foreach \x in {5,...,14}
+{
+    \draw[fill=green!50] (\x,-9) rectangle ++(1,1);
+}
+
+% Label for Bet 3
+\node[above] at (11.2,-8) {Bet 3: YES (\( b_1 = 10 \) units at \( p_1 = 0.5 \))};
+
+% Course Payout for Bet 3
+\def\xPartial{-3}  % Position of the partial square
+\draw (\xPartial,-9) rectangle ++(1,1);
+
+% Label for Course Payout of Bet 3
+\node[above] at (-4,-8) {Course Payout for Bet 3 (\( C_1 = 0.0 \) units)};
+
+
+\end{tikzpicture}
+\end{center}
+
+
+\newpage
+
+\section*{Normalization Factor Calculation for Third Transaction}
+
+\subsection*{Step Three: Calculate Normalization Factor}
```

Comment:
Just in general, how many significant figures do we use for shares, probabilities and normalization factors? Should a) state these explicitly b) keep them consistent